ci-operator/step-registry/gather/extra/gather-extra-commands.sh

```diff
@@ -137,6 +137,30 @@ for path in "${paths[@]}" ; do
   done < ${output_dir}.audit_logs_listing
 done
 
+
+echo "INFO: gathering quay tcpdump packet headers if present"
+output_dir="${ARTIFACT_DIR}/tcpdump/"
+mkdir -p "$output_dir"
+
+# Skip downloading of .terminating and .lock files.
+oc adm node-logs --role=worker --path="/tcpdump" | \
+grep -v ".terminating" | \
+grep -v ".lock" | \
+tee "${output_dir}.tcpdump_listing"
+
+cat "${output_dir}.tcpdump_listing"
+
+# The ${output_dir}.tcpdump_listing file contains lines with the node and filename
+# separated by a space.
+while IFS= read -r item; do
+node=$(echo $item |cut -d ' ' -f 1)
+fname=$(echo $item |cut -d ' ' -f 2)
+echo "INFO: Queueing download/gzip of /tcpdump/${fname} from ${node}";
+echo "INFO:   gziping to ${output_dir}/${node}-${fname}.gz";
+FILTER=gzip queue ${output_dir}/${node}-${fname}.gz oc --insecure-skip-tls-verify adm node-logs ${node} --path=/tcpdump/${fname}
```

Comment:
Interesting thing regarding my previous comment.  You save it to end with `.gz`, it gets uploaded to GCS as `.pcap` -- weird.

Reply:
Yup not sure what's going on there, I think I've seen that with the journal's we upload for nodes as well.

Anyhow probably best we can do for now, at least they are actually gzip'd.

Reply:
CI always eats the .gz suffix somewhere, I never figured out why but it's not specific to your code

Reply:
That is good to have confirmation of, thanks!